client/gutenberg/extensions/simple-payments/save.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+
+export default function Save( { attributes } ) {
+	const { paymentId } = attributes;
+	return paymentId ? <RawHTML>{ `[simple-payment id="${ paymentId }"]` }</RawHTML> : null;
```

Comment:
This seems to work without `RawHTML` just fine:
```js
return paymentId ? `[simple-payment id="${ paymentId }"]` : null;
```

Any reason to keep it?

Reply:
🔪 c44a37cfb4f3b40f074fc9fdb72648d7c49112eb